src/client.ts

```diff
@@ -14,144 +14,217 @@ import {
+   */
   async createCollection(
     collection: InscriptionCollectionCreateRequest
   ): Promise<InscriptionCollectionCreateResponse> {
     // modify normal json to valid form data for files
-    let plainObject = Object.assign({ ...collection });
-    let files = collection?.files;
+    let plainObject = Object.assign({ ...collection })
+    let files = collection?.files
     for (let index in files) {
-      let file: any = files[index];
-      let keys = Object.keys(file);
+      let file: any = files[index]
+      let keys = Object.keys(file)
       for (let key in keys) {
-        let propName = keys[key];
-        plainObject[`files[${index}][${propName}]`] = file[propName];
+        let propName = keys[key]
+        plainObject[`files[${index}][${propName}]`] = file[propName]
       }
     }
-    delete plainObject.files;
-    let data = qs.stringify(plainObject);
+    delete plainObject.files
+    let data = qs.stringify(plainObject)
     // modify normal json to valid form data for files
 
     let config = {
-      method: "post",
+      method: 'post',
       maxBodyLength: Infinity,
-      url: this.instanceV1.getUri() + "/collectioncreate",
+      url: this.instanceV1.getUri() + '/collectioncreate',
       headers: {
-        "Content-Type": "application/x-www-form-urlencoded",
+        'Content-Type': 'application/x-www-form-urlencoded',
       },
       data: data,
-    };
-    return axios.request(config);
+    }
+    return axios.request(config)
   }
 
+  /**
+   * Creates an order for a collection.
+   * @param {InscriptionCollectionOrderRequest} collectionOrder - The request object for creating the collection order.
+   * @returns {Promise<InscriptionCollectionOrderResponse>} A promise resolving with the created collection order response.
+   */
   async createCollectionOrder(
     collectionOrder: InscriptionCollectionOrderRequest
-  ): Promise<InscriptionOrder> {
-    return this.instanceV1.post(`/collectionorder`, collectionOrder);
+  ): Promise<InscriptionCollectionOrderResponse> {
+    return this.instanceV1.post(`/collectionorder`, collectionOrder)
   }
 
+  /**
+   * Creates an order for text inscription.
+   * @param {InscriptionTextOrderRequest} order - The request object for creating the text order.
+   * @returns {Promise<InscriptionOrder>} A promise resolving with the created text order.
+   */
   async createTextOrder(
     order: InscriptionTextOrderRequest
   ): Promise<InscriptionOrder> {
-    return this.instanceV1.post(`/textorder`, order);
+    return this.instanceV1.post(`/textorder`, order)
   }
 
+  /**
+   * Retrieves inventory information.
+   * @returns {Promise<InscriptionInventoryResponse[]>} A promise resolving with the inventory information.
+   */
   async getInventory(): Promise<InscriptionInventoryResponse[]> {
-    return this.instanceV1.get(`/inventory`);
+    return this.instanceV1.get(`/inventory`)
```

Comment:
I think we should agree on the same format. we should use `;` after each line so let's do it in this repo too. I don't know why this had to be deleted.
Can you add a common eslint file so we're not changing format with each PR. it's messy. we should stick to same formatting rules.